src/game.cpp

```diff
@@ -1873,6 +1933,18 @@ bool Game::startup(bool *kill,
 
 	smgr->getParameters()->setAttribute(scene::OBJ_LOADER_IGNORE_MATERIAL_FILES, true);
 
+	memset(&runData, 0, sizeof(runData));
```

Comment:
FWIW I prefer the aggregate initialization, doesn't require a separate function call.